app/components/examplePage1/examplePage1.html

```diff
@@ -1,11 +1,29 @@
 <div class="container">
 	<div class="row">
 		<div class="col-xs-12">
-			<h1>Example Page #1!</h1>
-			<h2>{{exampleContents1}}</h2>
+			<h1>Total Money For Election</h1>
+			<h2> {{total | currency}} </h2>
+			<div ng-repeat="item in data">
+			  <h2 ng-show="item['entity_cd'] == 'NIT'">
```

Comment:
You can use the dot properties here to

    <h2 ng-show="item.entity_cd == 'NIT'">